kubernetes/assets/recommended_monitors/monitor_pod_crashloopbackoff.json

```diff
@@ -0,0 +1,27 @@
+{
+	"name": "[kubernetes] Pod {{pod_name.name}} is CrashloopBackOff on namespace {{kube_namespace.name}}",
+	"type": "query alert",
+	"query": "avg(last_30m):default(avg:kubernetes.containers.state.waiting{reason:crashloopbackoff} by {kube_namespace,pod_name}, 0) >= 1",
```

Comment:
1. using avg (*)
2. last 30m might be too big a time frame.
3. do we need the default function in the query?

Reply:
agg time depends on customer workloads.